pom.xml

```diff
@@ -46,6 +46,13 @@
         <scala.version>2.12</scala.version>
         <spark.version>3.3.0</spark.version>
         <jackson.core.version>2.13.3</jackson.core.version>
+        <argLine>
+            --add-exports java.base/sun.nio.ch=ALL-UNNAMED
+            --add-opens java.base/java.nio=ALL-UNNAMED
+            --add-opens java.base/java.io=ALL-UNNAMED
+            --add-opens java.base/java.util=ALL-UNNAMED
+            --add-opens java.base/java.lang.invoke=ALL-UNNAMED
+        </argLine>
```

Comment:
@lbergelson I know it seems sketchy, but I think this is just what you have to do. You should see the list for GATK....

We might want to extend this list to just automatically include all the [ones used by spark](https://github.com/apache/spark/blob/v3.3.0/launcher/src/main/java/org/apache/spark/launcher/JavaModuleOptions.java#L21) (thats what I did for GATK).